include/umf/providers/provider_os_memory.h

```diff
@@ -63,13 +63,12 @@ typedef struct umf_os_memory_provider_params_t {
     unsigned protection;
 
     // NUMA config
-    /// Points to a bit mask of nodes containing up to maxnode bits, depending on
-    /// selected numa_mode newly allocated memory will be bound to those nodes
-    unsigned long *nodemask;
-    /// Max number of bits in nodemask
-    unsigned long maxnode;
+    size_t *numa_list;      // ordered list of numa nodes
+    unsigned numa_list_len; // length of numa_list
+
     /// Describes how nodemask is interpreted
     umf_numa_mode_t numa_mode;
+    size_t part_size;
```

Comment:
Should we also introduce a new policy, like UMF_NUMA_MODE_BLOCKED? Or is the idea to support part_size for different policies?

Reply:
This line of change is an accident it should be included in next patches. Anyway i plan to change implementation of some policies in os provider in next patches.  i.e interleaving will use part_size (what requires implementation of interlining on our side)